includes/class-multisite-directory-taxonomy.php

```diff
@@ -133,4 +135,32 @@ public function saveTermGeo ($term_id) {
         }
     }
 
+    /**
+     * Adds a dropdown box to filter directory entries by category in the admin listing
+     *
+     * @link http://www.wpcustoms.net/snippets/admin-filter-for-custom-taxonomies/
```

Comment:
If you're going to link a docblock somewhere, choose a link that provides additional information beyond just duplicating the same code. For instance, the WP code reference for the hook used. Otherwise, remove the link.